src/apps/ociobakelut/main.cpp

```diff
@@ -162,54 +183,90 @@ int main (int argc, const char* argv[])
+            {
+                std::cerr << "\nERROR: --shaperspace is not allowed when using --lut\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
 
-        OCIO::ConfigRcPtr editableConfig = OCIO::Config::Create();
+            OCIO::ConfigRcPtr editableConfig = OCIO::Config::Create();
 
-        OCIO::ColorSpaceRcPtr inputColorSpace = OCIO::ColorSpace::Create();
-        inputspace = "RawInput";
-        inputColorSpace->setName(inputspace.c_str());
-        editableConfig->addColorSpace(inputColorSpace);
+            OCIO::ColorSpaceRcPtr inputColorSpace = OCIO::ColorSpace::Create();
+            inputspace = "RawInput";
+            inputColorSpace->setName(inputspace.c_str());
+            editableConfig->addColorSpace(inputColorSpace);
 
-        OCIO::ColorSpaceRcPtr outputColorSpace = OCIO::ColorSpace::Create();
-        outputspace = "ProcessedOutput";
-        outputColorSpace->setName(outputspace.c_str());
+            OCIO::ColorSpaceRcPtr outputColorSpace = OCIO::ColorSpace::Create();
+            outputspace = "ProcessedOutput";
+            outputColorSpace->setName(outputspace.c_str());
 
-        outputColorSpace->setTransform(groupTransform,
-            OCIO::COLORSPACE_DIR_FROM_REFERENCE);
+            outputColorSpace->setTransform(groupTransform,
+                OCIO::COLORSPACE_DIR_FROM_REFERENCE);
 
-        if(verbose)
-        {
-            std::cout << "[OpenColorIO DEBUG]: Specified Transform:";
-            std::cout << *groupTransform;
-            std::cout << "\n";
+            if(verbose)
+            {
+                std::cout << "[OpenColorIO DEBUG]: Specified Transform:";
+                std::cout << *groupTransform;
+                std::cout << "\n";
+            }
+
+            editableConfig->addColorSpace(outputColorSpace);
+            config = editableConfig;
         }
+        else
+        {
+            if(inputspace.empty())
+            {
+                std::cerr << "\nERROR: --inputspace is required when using --view\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
+            if(!outputspace.empty())
+            {
+                std::cerr << "\nERROR: --outputspace is not allowed when using --view\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
+
+            OCIO::ConfigRcPtr editableConfig = OCIO::Config::Create();
+            OCIO::ColorSpaceRcPtr inputColorSpace = OCIO::ColorSpace::Create();
+            inputColorSpace->setName(inputspace.c_str());
+            editableConfig->addColorSpace(inputColorSpace);
+
+            OCIO::ColorSpaceRcPtr outputColorSpace = OCIO::ColorSpace::Create();
+            outputspace = "ProcessedOutput";
+            outputColorSpace->setName(outputspace.c_str());
+
+            auto t = OCIO::DisplayViewTransform::Create();
```

Comment:
Here I wanted to get the `DisplayViewTransformRcPtr` from `groupTransform` using `groupTransform->getTransform(0)` but I wasn't able to cast it correctly. If someone is ready to help me with this, I'd be glad.